docs/src/rules/no-multi-assign.md

```diff
@@ -27,7 +27,7 @@ Examples of **incorrect** code for this rule:
 ```js
 /*eslint no-multi-assign: "error"*/
 
-var a = b = c = 5;
+const a = b = c = 5;
```

Comment:
```suggestion
let a = b = c = 5;
```
`a` is reassigned in following example `a = b = "quux"`.

Reply:
thanks